django/core/management/commands/runserver.py

```diff
@@ -181,10 +187,21 @@ def on_bind(self, server_port):
 
         now = datetime.now().strftime("%B %d, %Y - %X")
         version = self.get_version()
+
         print(
             f"{now}\n"
             f"Django version {version}, using settings {settings.SETTINGS_MODULE!r}\n"
             f"Starting development server at {self.protocol}://{addr}:{server_port}/\n"
             f"Quit the server with {quit_command}.",
             file=self.stdout,
         )
+        if not self.hide_production_warning_message:
```

Comment:
Can we avoid showing this warning in local development environment?

It's for sure useful in production environment, but it adds noise locally on your development machine.

At least, we can define an environment variable to hide the warning without passing an argument to `runserver`.